components/web/php/php-7_3/Makefile

```diff
@@ -0,0 +1,218 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"). You may
+# only use this file in accordance with the terms of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source. A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2013, EveryCity Ltd. All rights reserved.
+#
+include ../../../../make-rules/shared-macros.mk
+
+COMPONENT_NAME=		php
+COMPONENT_MJR_VERSION=	7.3
+COMPONENT_MNR_VERSION=	4
+COMPONENT_VERSION=	$(COMPONENT_MJR_VERSION).$(COMPONENT_MNR_VERSION)
+COMPONENT_SRC=		$(COMPONENT_NAME)-$(COMPONENT_VERSION)
+COMPONENT_ARCHIVE=	$(COMPONENT_SRC).tar.xz
+COMPONENT_ARCHIVE_HASH=	sha256:6fe79fa1f8655f98ef6708cde8751299796d6c1e225081011f4104625b923b83
+COMPONENT_ARCHIVE_URL=	http://uk1.php.net/get/$(COMPONENT_ARCHIVE)/from/this/mirror
+COMPONENT_SIG_URL=	http://uk1.php.net/get/$(COMPONENT_ARCHIVE).asc/from/this/mirror
+COMPONENT_LICENSE=	PHP License
+COMPONENT_LICENSE_FILE=	LICENSE
+
+include $(WS_MAKE_RULES)/prep.mk
+include $(WS_MAKE_RULES)/configure.mk
+include $(WS_MAKE_RULES)/ips.mk
+
+# Apache Paths
+AP_PREFIX=/usr/apache2/2.4
+AP_SYSCONFDIR=/etc/apache2/2.4
+
+# PHP Paths
+PHP_PREFIX=/usr/php/$(COMPONENT_MJR_VERSION)
+PHP_SYSCONFDIR=/etc/php/$(COMPONENT_MJR_VERSION)
+PHP_DATADIR=/var/php/$(COMPONENT_MJR_VERSION)
+
+CFLAGS			= $(CC_BITS) -D_POSIX_PTHREAD_SEMANTICS
+CPPFLAGS		+= -I/usr/include/openldap
+CPPFLAGS		+=      -I$(MYSQL_INCDIR)
+CFLAGS			+=	-I$(MYSQL_INCDIR)
+LDFLAGS		+=	-L$(MYSQL_LIBDIR) -R$(MYSQL_LIBDIR)
+
+# build with the system default libjpeg
+CFLAGS		+=	$(JPEG_CPPFLAGS) $(JPEG_CFLAGS)
+CXXFLAGS	+=	$(JPEG_CPPFLAGS) $(JPEG_CXXFLAGS)
+CPPFLAGS	+=	$(JPEG_CPPFLAGS)
+LDFLAGS		+=	$(JPEG_LDFLAGS)
+
+COMPONENT_PREP_ACTION =        ( cd $(@D) && autoconf )
+
+# Configure Options
+CONFIGURE_OPTIONS  =	--prefix=$(PHP_PREFIX)
+CONFIGURE_OPTIONS  +=	--bindir=$(PHP_PREFIX)/bin
+CONFIGURE_OPTIONS  +=	--sbindir=$(PHP_PREFIX)/bin
```

Comment:
`sbin`?

Reply:
no, we don't have /usr/php/X.Y/sbin, just bin.